### ISSUE TITLE
Reproduce behavior of `load_example` from `dscr`

### ISSUE THREAD

**Issue by gaow** [stephenslab/dsc#120]
We may want to add for `dscrutils` something like `dscr::load_example`. Currently I have something like this implemented:

```
[GW] cd one_sample_location/first_investigation/t/
[GW] ls
t_1.rds
[GW] echo a | dsc-query t_1.rds -o interactive 
INFO: Loading database ...
INFO: Data dumped to text files interactive.out and interactive.script.
```

It basically saves the script that generates the output , and dump the output in plain text. But it would be better to have `dscrutils` handle it.

**Comment by pcarbo** [comment]
@gaow Did Matthew ask for this? Running `cat(readRDS("t_1.rds")$DSC_DEBUG$script)` in R seems easy enough; do you think we really need anothe function to do this?

**Comment by gaow** [comment]
Yes, I'm just documenting a request from Matthew. We'll still have to see what is the best way to combine it with `dscquery()`.

**Comment by gaow** [comment]
So @jhsiao999 also asked about extracting runtime sessioninfo easily -- currently it is same way as how `script` is stored. We can think of an interface for that info.
